packages/terra-consumer-icon/src/IconBase.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
```

Comment:
good point. It make sense - [updated](https://github.com/cerner/terra-consumer/pull/7/commits/8afa0d91915152b9123d99297bca959a8b410a90)